### PR TITLE
Bugfix

### LINKED ISSUE #59: Unable to use TaskElementsValueUtils.addElementsValue
Use TaskElementsValueUtils.addElementsValue() throws an error (unmodifiable collectionexception). For example in SaveTaskElementRequestUtils

### PR DESCRIPTION
**Pull request by 7eben**
Close #59

### CODE REVIEW

File: kuflow-rest/pom.xml [hunk 1]
```diff
@@ -30,7 +30,7 @@
   <parent>
     <groupId>com.kuflow</groupId>
     <artifactId>kuflow-sdk</artifactId>
-    <version>2.6.1-SNAPSHOT</version>
+    <version>2.6.1</version>
```
Comment (jrpedrianes) [inline kuflow-rest/pom.xml:33]: Before merge, I think that this must be a SNAPSHOT

**Review by jrpedrianes — CHANGES_REQUESTED** [review]

File: kuflow-rest/src/test/java/com/kuflow/rest/util/TaskElementValueAccessorMock.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import com.kuflow.rest.model.TaskElementValue;
+import java.util.List;
+
+public class TaskElementValueAccessorMock implements TaskElementValueAccessor {
+
+    private List<TaskElementValue> elementValues;
+
+    @Override
+    public List<TaskElementValue> getElementValues() {
+        return this.elementValues;
```
Comment (jrpedrianes) [inline kuflow-rest/src/test/java/com/kuflow/rest/util/TaskElementValueAccessorMock.java:34]:
```suggestion
        return Collections.unmodifiableList(this.elementValues);
```

File: kuflow-rest/src/test/java/com/kuflow/rest/util/TaskElementValueAccessorMock.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    }
+
+    @Override
+    public void setElementValues(List<TaskElementValue> elementValues) {
+        this.elementValues = elementValues;
```
Comment (jrpedrianes) [inline kuflow-rest/src/test/java/com/kuflow/rest/util/TaskElementValueAccessorMock.java:39]:
```suggestion
        this.elementValues.clear()
        if (elementValues != null) {
            this.elementValues.addAll(elementValues);
        }
```

**Review by jrpedrianes — CHANGES_REQUESTED** [review]

File: kuflow-rest/src/test/java/com/kuflow/rest/util/TaskElementValueAccessorMock.java (hunk 3 above)
Comment (7eben) [reply kuflow-rest/src/test/java/com/kuflow/rest/util/TaskElementValueAccessorMock.java]: I put Unmodifiable in the mock to simulate the real world

File: kuflow-rest/src/test/java/com/kuflow/rest/util/TaskElementValueAccessorMock.java (hunk 2 above)
Comment (7eben) [reply kuflow-rest/src/test/java/com/kuflow/rest/util/TaskElementValueAccessorMock.java:35]: I put Unmodifiable in "set" of the mock to simulate the real world

Comment (jrpedrianes) [reply kuflow-rest/src/test/java/com/kuflow/rest/util/TaskElementValueAccessorMock.java]: Ok, but I think it is better always for the list to be unmodifiable and immutable.
Seen properly your code, maybe is better to do something like this:

```
public List<TaskElementValue> getElementValues() {
    if (this.elementValues == null) {
        return null;
     }
     return Collections.unmodifiableList(this.elementValues);
}

public void setElementValues(List<TaskElementValue> elementValues) {
        if (elementValues == null) {
              this.elementValues = null;
        }  else {
              this.elementValues = new LinkedList(elementValues);
        }
}
```

**Review by jrpedrianes — CHANGES_REQUESTED** [review]

**Review by jrpedrianes — APPROVED** [review]

**Pull request merged**